src/Compilers/CSharp/Portable/Lowering/MethodToClassRewriter.cs

```diff
@@ -194,7 +194,7 @@ public override BoundNode VisitUsingStatement(BoundUsingStatement node)
             BoundExpression expressionOpt = (BoundExpression)this.Visit(node.ExpressionOpt);
             BoundStatement body = (BoundStatement)this.Visit(node.Body);
             Conversion disposableConversion = RewriteConversion(node.IDisposableConversion);
-            return node.Update(newLocals, declarationsOpt, expressionOpt, disposableConversion, body);
+            return node.Update(newLocals, declarationsOpt, expressionOpt, disposableConversion, body, awaitOpt: null);
```

Comment:
>awaitOpt: null [](start = 102, length = 14)

Let's assert that awaitOpt was null in the original node and that it was not of await kind. I am curious if this method is reachable at all, it looks like we should not have any BoundUsingStatement nodes in the tree after the LocalRewriter.

Reply:
Confirmed that this rewriter is only involved after local rewriting and that using bound nodes cannot survive local rewriting (VisitUsingStatement rewrites it to try/catch and no other local rewriter case produces a using). Added assertion.